catalogs/catalog-hive/src/main/java/com/datastrato/gravitino/catalog/hive/HiveCatalogOperations.java

```diff
@@ -157,6 +265,16 @@ public void close() {
       clientPool.close();
       clientPool = null;
     }
+
+    if (refreshScheduledExecutor != null) {
+      refreshScheduledExecutor.shutdown();
+      refreshScheduledExecutor = null;
+    }
+
+    File keyTabFile = new File(String.format("/tmp/gravitino-%s-keytab", entity.id()));
```

Comment:
It is not safe to put keytab in /tmp folder, /tmp folder will be cleaned by the administrator. You'd better choose a new folder.

Reply:
Fixed.